src/webapi/html_elements/media.rs

```diff
@@ -0,0 +1,183 @@
+use webcore::value::Reference;
+use webcore::try_from::TryInto;
+use webapi::event_target::{IEventTarget, EventTarget};
+use webapi::node::{INode, Node};
+use webapi::element::{IElement, Element};
+use webapi::html_element::{IHtmlElement, HtmlElement};
+
+/// The readiness state of a media element.
+///
+/// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/readyState)
+#[derive(Copy, Clone, Debug, PartialEq, Eq, PartialOrd, Ord)]
+pub enum ReadyState {
```

Comment:
`ReadyState` -> `MediaReadyState` maybe? (We have way to many `ReadyState`s, so I'd like them to all have unique names to reduce confusion.)

Reply:
Great idea! I'll rename it.